test_suite/cloud/test_aws.py

```diff
@@ -117,7 +125,11 @@ def test_auditd(self, host):
             auditd_service).is_running, f'{auditd_service} expected to be running'
 
         system_release = version.parse(host.system_info.release)
-        if system_release >= version.parse('8.6'):
+        if system_release >= version.parse('9.4'):
+            checksums = checksums_by_version['9.4+']
+        elif system_release >= version.parse('8.10'):
```

Comment:
We'll need to add something like 
```suggestion
        elif system_release >= version.parse('8.10') and system_release < version.parse('9.0'):
``` 
(syntax is probably wrong) because otherwise it's matching the 9.1 GA which is not what we want, we want just rhel-8 matched.